core_api/src/modules/patient/patient.resolver.ts

```diff
@@ -13,4 +14,19 @@ export default class PatientResolver {
       }
     });
   }
+
+  // TODO : rescrtict access to role === doctor | secretary
+  // TODO : limit the number of results ?
+
+  // needed to browse patients by their first or lastname, case insensitive
+  @Query(() => [Patient])
+  async patients(@Arg('search') search: string) {
+    if (search === '' || search === ' ') return [];
+    return await Patient.find({
+      where: [
+        { firstname: ILike(`${search}%`) },
```

Comment:
It looks (?) from the TypeORM docs like this will do an AND on the two where conditions, not an OR: https://orkhan.gitbook.io/typeorm/docs/find-options